apps/src/codegen.js

```diff
@@ -15,6 +15,8 @@ exports.ForStatementMode = {
   UPDATE: 3
 };
 
+exports.asyncFunctionList = [];
```

Comment:
Let's either add a descriptive comment describing how this exported array is to be used, or do something to make it obvious that it's private-ish / shouldn't be used directly.